doc/source/features/online-schema-changes.rst

```diff
@@ -0,0 +1,82 @@
+
+.. list-table::
+    :widths: 20 60
+    :header-rows: 1
+
+    * - Method 
+      - Description
+    * - `TOI`
+      - Total Order Isolation
+        
+        The default DDL replication method for the :ref:`wsrep osu method` variable. The isolation of the DDL statement guarantees consistency.
+        
+        The node detects a DDL statement in the Parser stage and sends the statement to all nodes. All nodes wait for any earlier transactions to commit, and then execute the statement.
+        
+        All other writes must wait until the DDL statement is executed. While a DDL statement is running, any long-running transactions in progress and using the same resource receive a deadlock error at commit and are rolled back. There is no way to verify if individual nodes have succeeded in processing the DDL statement. 
+    * - `RSU`
+      - Rolling  Schema Upgrade
+
+        A DDL statement are processed on each nod locally. This action guarantees high availability. The node desynchronizes with the cluster and disables flow control while executing the DDL statement. The rest of the cluster is not affected. After the statement execution, the node applies delayed events and synchronizes with the cluster. 
+
+        The cluster continues to operate, although during the process, some nodes may be updated and some nodes may have the older schema structure.
+
+        For this method, the ``gcache`` must be large enough to store the data for the duration of the DDL change. 
+    * - `NBO`
+      - Non-Blocking Operation
+
+        This method is similar to ``TOI``. All nodes acquire metadata locks on each table or schema for any open transactions and do not release those locks until the transaction ends. The DDL statements execute on all nodes after the transactions have completed. This method avoids the TOI issue when a long-running DDL blocks other updates in the cluster.
+
+        For this method, the ``gcache`` must be large enough to store the data for duration of the DDL statement. 
+
+Using the NBO method
+----------------------
+
+You set the method for online schema changes by either using the ``wsrep_OSU_method`` variable in ``my.cnf`` or  using the ``SET`` statement in a MySQL client. 
+
+To set the variable in the my.cnf file, do the following:
+
+.. sourcecode:: text
+
+    wsrep_osu_method=NBO
+
+Restart the server. 
+
+To dynamically set the ``NBO`` method in the client, run the following statement:
+
+.. sourcecode:: mysql
+
+    SET GLOBAL wsrep_OSU_method='NBO';
+
+
+Limitations
+-----------------
+
+The following items are the limitations of using ``NBO``:
+
+* Write attempts are blocked until the ``ALTER`` statement completes. If the DDL statement is ``ALTER TABLE ... LOCK=SHARED``, read operations are allowed. If the statement is ``ALTER TABLE ... LOCK=EXCLUSIVE``, read operations are not allowed.
```

Comment:
This just describes how MySQL works, what's the PXC/NBO limitation here?

Reply:
Changed to say NBO OSC